Random Users/Operations/FetchImageOperation.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  FetchImageOperation.swift
+//  Random Users
+//
+//  Created by Norlan Tibanear on 9/11/20.
+//  Copyright © 2020 Erica Sadun. All rights reserved.
+//
+
+import Foundation
+
+
+class FetchImageOperation: ConcurrentOperation {
+    
+    var user: UserResults
+    var imageData: Data?
+    
+    private var loadImageData: URLSessionTask?
+    
+    init(user: UserResults) {
+        self.user = user
+    }
+    
+    override func start() {
+        
+        state = .isExecuting
+        
+        defer { state = .isFinished }
```

Comment:
This should be in the data task completion closure! Having it here is incorrect.